src/Sylius/Behat/Context/Api/Admin/NotificationContext.php

```diff
@@ -1,53 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sylius package.
- *
- * (c) Paweł Jędrzejewski
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-declare(strict_types=1);
-
-namespace Sylius\Behat\Context\Api\Admin;
-
-use Behat\Behat\Context\Context;
-use Sylius\Behat\Client\ApiClientInterface;
-use Webmozart\Assert\Assert;
-
-final class NotificationContext implements Context
```

Comment:
Maybe it wasn't a bad idea to have these checks in one place?

Reply:
I agree, but I have no idea, how it should be implemented since we have api client per resource 😞

Reply:
It wasn't, but we also would have to register this context with each api_platform_client, as there is a requirement to have the `resource` property set. In theory - it doesn't matter, as we only check the status. But we still need to get the response from the client :/ So in a result - it's worth to be refactored, but probably not now 🖖